pkg/kubelet/cri/streaming/server.go

```diff
@@ -34,7 +34,7 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 	remotecommandconsts "k8s.io/apimachinery/pkg/util/remotecommand"
 	"k8s.io/client-go/tools/remotecommand"
-	runtimeapi "k8s.io/cri-api/pkg/apis/runtime/v1"
```

Comment:
I'm not sure about the changes in this package... tests and type checking is complaining about it

Reply:
yes, I realized it minutes before it was too late and needed to go to get some sleep